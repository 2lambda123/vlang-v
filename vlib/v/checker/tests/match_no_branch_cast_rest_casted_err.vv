fn main () {
	name := 'b'
	nb := match name {
		'a' { u8(10) }
		'b' { u8(12) }
		else { 0 }
	}

	println(nb)
}
