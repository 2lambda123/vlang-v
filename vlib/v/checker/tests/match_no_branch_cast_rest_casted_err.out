vlib/v/checker/tests/match_no_branch_cast_rest_casted_err.vv:6:10: error: expected an explicit cast type of `u8` from first branch
    4 |         'a' { u8(10) }
    5 |         'b' { u8(12) }
    6 |         else { 0 }
      |                ^
    7 |     }
    8 |
